### PR TITLE
Setup Travis-CI

### PR DESCRIPTION
**Pull request by Gigabyte-Giant**
Added a `.travis.yml` file and configured it to use NodeJS and Gulp.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+language: node_js
+node_js:
```
Comment (Gigabyte-Giant) [inline .travis.yml]: @JavascriptFTW Do you have any preference on which version of NodeJS we should use?

Comment (JavascriptFTW) [inline .travis.yml]: Not particularly.

**Comment by JavascriptFTW** [comment]
Looks good. Mind rebasing before I merge?

**Comment by JavascriptFTW** [comment]
Looks good! Thanks for the PR!

**Pull request merged**